astronomer/providers/amazon/aws/operators/redshift_sql.py

```diff
@@ -1,7 +1,14 @@
 from typing import Any, cast
 
 from airflow.exceptions import AirflowException
-from airflow.providers.amazon.aws.operators.redshift_sql import RedshiftSQLOperator
+
+try:
+    from airflow.providers.amazon.aws.operators.redshift_sql import RedshiftSQLOperator
+except ImportError:  # pragma: no cover
+    # For apache-airflow-providers-snowflake > 6.0.0
```

Comment:
this should be `apache-airflow-providers-amazon`?

Reply:
Yes fixed